micro_brewery/data/product_product.xml

```diff
@@ -149,63 +149,83 @@
+        <field name="uom_id" ref="uom_uom_31"/>
+        <field name="uom_po_id" ref="uom_uom_31"/>
         <field name="image_1920" type="base64" file="micro_brewery/static/src/binary/product_template/30-image_1920"/>
     </record>
     <record id="product_product_35" model="product.product">
         <field name="name">Brown Beer (25cl)</field>
         <field name="list_price">2.5</field>
-        <field name="categ_id" ref="product_category_6"/>
+        <field name="categ_id" ref="product_category_103"/>
         <field name="purchase_ok" eval="False"/>
         <field name="available_in_pos" eval="True"/>
         <field name="pos_categ_ids" eval="[(6, 0, [ref('pos_category_2')])]"/>
-        <field name="x_beer" ref="product_template_21"/>
         <field name="base_unit_id" ref="base_unit_litre"/>
+        <field name="base_unit_count">0.25</field>
+        <field name="uom_id" ref="uom_uom_31"/>
+        <field name="uom_po_id" ref="uom_uom_31"/>
         <field name="image_1920" type="base64" file="micro_brewery/static/src/binary/product_template/32-image_1920"/>
     </record>
     <record id="product_product_36" model="product.product">
         <field name="name">Brown Beer (50cl)</field>
         <field name="list_price">4.5</field>
-        <field name="categ_id" ref="product_category_6"/>
+        <field name="categ_id" ref="product_category_103"/>
         <field name="purchase_ok" eval="False"/>
         <field name="available_in_pos" eval="True"/>
         <field name="pos_categ_ids" eval="[(6, 0, [ref('pos_category_2')])]"/>
-        <field name="x_beer" ref="product_template_21"/>
         <field name="base_unit_id" ref="base_unit_litre"/>
+        <field name="base_unit_count">0.5</field>
+        <field name="uom_id" ref="uom_uom_30"/>
+        <field name="uom_po_id" ref="uom_uom_30"/>
         <field name="image_1920" type="base64" file="micro_brewery/static/src/binary/product_template/33-image_1920"/>
     </record>
     <record id="product_product_37" model="product.product">
         <field name="name">Lager Beer (50cl)</field>
         <field name="list_price">3.8</field>
-        <field name="categ_id" ref="product_category_6"/>
+        <field name="categ_id" ref="product_category_104"/>
         <field name="purchase_ok" eval="False"/>
         <field name="available_in_pos" eval="True"/>
         <field name="pos_categ_ids" eval="[(6, 0, [ref('pos_category_2')])]"/>
-        <field name="x_beer" ref="product_template_22"/>
         <field name="base_unit_id" ref="base_unit_litre"/>
+        <field name="base_unit_count">0.5</field>
+        <field name="uom_id" ref="uom_uom_30"/>
+        <field name="uom_po_id" ref="uom_uom_30"/>
         <field name="image_1920" type="base64" file="micro_brewery/static/src/binary/product_template/34-image_1920"/>
     </record>
+    <record id="product_product_29" model="product.template" context="{'create_product_product': False}">
```

Comment:
Why a product template here?

Reply:
I assume the fix was not to move it to the template, but instead declare a product, no? (just like other 50cl beers)